cranelift/codegen/src/isa/aarch64/inst/emit_tests.rs

```diff
@@ -1375,6 +1375,69 @@ fn test_aarch64_binemit() {
         "cls x21, x16",
     ));
 
+    insns.push((
+        Inst::BitRR {
+            op: BitOp::Rev16,
+            size: OperandSize::Size64,
+            rd: writable_xreg(2),
+            rn: xreg(11),
+        },
+        // expect:
+        // 1101_1010_110 | 0_0000_0000_01 | 01_011 | 0_0010
+        // which is little endian:
+        // 0110_0010_0000_0101_1100_0000_1101_1010
```

Comment:
The little-endian bit literal here is kind of confusing (it's little-endian but MSB first); if we need an explanatory comment can we convert the above to a u32 (`0xdac00562`) or maybe just remove this comment?

Reply:
These comments were just to help me out when writing the test cases; I've removed them now